nbbuild/antsrc/org/netbeans/nbbuild/extlibs/CreateDependencies.java

```diff
@@ -107,7 +107,7 @@ public void setSourceDependencies(boolean sourceDependencies) {
 
                 Map<String, Map<String, Set<String>>> dependenciesDesc = new LinkedHashMap<>();
                 dependenciesDesc.put("Runtime dependencies:", origin2TextsRuntime);
-                if (sourceDependencies) {
+                if (sourceDependencies && /*don't print the compile-time dependencies*/false) {
```

Comment:
It was not explicitly part of the objection, but when the list of compile-time dependencies is incomplete (at least to some degree), it seemed better to avoid it at all.

Reply:
Is the list incomplete? My understanding of the change-set was, that the license info is removed, the reference is still in place. Anyway - I have no strong feelings about this - I just noticed it. The changeset can IMHO be applied as is (maybe lets wait till the VOTE Emilian started is done).

Reply:
> maybe lets wait till the VOTE Emilian started is done

Voting has finished. We are free to merge.